contracts/BucketAuction.sol

```diff
@@ -41,10 +43,9 @@ contract BucketAuction is IBucketAuction, ERC721M {
             maxMintableSupply,
             globalWalletLimit,
             cosigner,
-            /* timestampExpirySeconds= */
-            300,
-            /* mintCurrency= */
-            address(0)
+            timestampExpirySeconds,
+            address(0),
+            crossmintAddress
```

Comment:
Let's set crossmintAddress to `address(0)` for in BucketAuction and DutchAuction (and their variance). I don't think we should support crossmint in auctions.